### PR TITLE
v1.0

### PR DESCRIPTION
**Pull request by dgeb**
After two years, four release candidates, hundreds of contributions, and untold hours of discussions, we're finally ready to declare the JSON API specification stable!

JSON API v1.0 marks the beginning of a new phase in the life of this specification. From this point forward, the base specification will remain backwards compatible and will only change additively.

---

Thanks so much to this wonderful community for discussing, contributing to, and implementing this specification over this long journey. @wycats, @steveklabnik, @tkellen and I could not have done this without your help!

### CODE REVIEW

**Comment by kellysutton** [comment]
:+1:x100

**Comment by tkellen** [comment]
:shipit: :tada:

File: faq/index.md [hunk 1]
```diff
@@ -3,11 +3,15 @@ layout: page
...
 ---
 
-## Why is JSON API not versioned? <a href="#why-is-json-api-not-versioned" id="why-is-json-api-not-versioned" class="headerlink"></a>
+## What is the meaning of JSON API's version? <a href="#what-is-the-meaning-of-json-apis-version" id="what-is-the-meaning-of-json-apis-version" class="headerlink"></a>
 
-Once JSON API is stable, it will always be backwards compatible using a _never
-remove, only add_ strategy.
-[#46](https://github.com/json-api/json-api/issues/46)
+Now that JSON API has reached a stable version 1.0, it will always be
+backwards compatible using a _never remove, only add_ strategy.
```
Comment (krainboltgreene) [inline faq/index.md:9]: What's the point of a major version number with this strategy?

Comment (nonsensery) [inline faq/index.md:9]: I'm interpreting this as "we're going to follow semver, and we don't ever plan on making a version 2.0."

**Comment by barelyknown** [comment]
Outstanding! What a huge accomplishment for the community. Big thanks to the core team and everyone that contributed.

**Comment by shingara** [comment]
:+1:

**Comment by lgebhardt** [comment]
:+1: Nice work!

**Comment by steveklabnik** [comment]
:confetti_ball:

**Comment by wayneashleyberry** [comment]
Woohoo!

**Comment by jede** [comment]
YAY!! :tada:

**Comment by catesandrew** [comment]
:+1:

**Comment by alexaivars** [comment]
Best Friday commit EVER!

**Comment by cibernox** [comment]
:confetti_ball:

**Comment by stevenharman** [comment]
:tada: :sparkles: 

MUCH EXCITE!

**Comment by iheanyi** [comment]
:+1: :+1:

**Comment by achan** [comment]
Great job everybody!

**Comment by zeppelin** [comment]
:fireworks:

**Comment by ethanresnick** [comment]
YAYAY! :+1:

**Comment by HeroicEric** [comment]
:boom:

**Comment by iho** [comment]
:cocktail:

**Comment by patrykorwat** [comment]
:sparkles: Great! :sparkles:

**Comment by cziegenberg** [comment]
🍻 Prost!

**Comment by hhware** [comment]
Congratulations, and big thanks to the editors for driving this!

**Comment by mahmoudimus** [comment]
:confetti_ball: grats! :+1:

**Comment by soulim** [comment]
Thank you! :smiley::+1:

**Comment by rahult** [comment]
Well done :beers:

**Comment by pixelhandler** [comment]
Super awesome to have a 1.0 spec. Already putting into use thanks so much!

**Comment by ballPointPenguin** [comment]
:rainbow: :honey_pot: :boat: :computer:

**Comment by SeyZ** [comment]
Awesome :+1:

**Comment by mjallday** [comment]
:+1: w00t!

**Pull request merged**